website_sale_b2b/models/sale_order.py

```diff
@@ -51,7 +51,11 @@ def action_quotation_send(self):
 class SaleOrderLine(models.Model):
     _inherit = "sale.order.line"
 
-    name = fields.Text(compute="_recompute_name", store=True)
+    name = fields.Text(
+        compute="_recompute_name",
+        inverse=lambda self: None,  # Make it updatable through the UI despite compute
```

Comment:
FYI, in recent version computed field can be set as "readonly=False". but don't know when it begin.